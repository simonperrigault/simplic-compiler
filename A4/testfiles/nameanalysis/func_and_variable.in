int b() {
    return 0;
}

int main() {
    int b = 2;
    int a = b(); // error but seen in next assignments
    return 0;
}