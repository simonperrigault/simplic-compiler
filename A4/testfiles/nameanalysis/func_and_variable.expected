Error at line 7: symbol 'b' is a function call but declared as a variable
